py_ecc/bls/g2_primatives.py

```diff
@@ -1,4 +1,3 @@
 # flake8: noqa
 # `g2_primatives.py` was a typo of `g2_primitives.py`
 # Make an alias for backward compatibility
-from py_ecc.bls.g2_primitives import *
```

Comment:
If we're deleting this line/making a breaking change to the library, I think this file can go?

Reply:
removed & updated refs